docs/data/introduction/licensing/licensing.md

```diff
@@ -290,15 +290,26 @@ This happens if you try to use `DataGridPremium` with a license key for the Pro
 To solve the issue, you can [upgrade your plan](https://mui.com/r/x-get-license/?scope=premium) from Pro to Premium.
 Or if you didn\'t intend to use Premium features, you can replace the import of `@mui/x-data-grid-premium` with `@mui/x-data-grid-pro`.
 
-### 5. Invalid license key
+### 5. Product not covered by the license key
+
+This error indicates that the product you are trying to use is not covered by the license key.
+This happens if you try to use `ChartsPro` or `TreeViewPro` with a license that does not cover these products.
+
+To solve the issue, you can [purchase a license](https://mui.com/r/x-get-license/) that covers the product you are trying to use.
```

Comment:
```suggestion
To solve the issue, please consider an earlier renewal of the [license](https://mui.com/r/x-get-license/).

You might be eligible for a discount if you have an active Pro license.
Contact [sales@mui.com](mailto:sales@mui.com?subject=My%20upgrade%20discount) for additional information.
```

Reply:
I'm phrasing this as a renewal, because for those users that were not using the cap (less than 10 seats), they can basically renew for the same price they purchased before, and they'll get a discount to compensate the "earlier" aspect of the renewal.